cmd/agent/app/check.go

```diff
@@ -27,7 +27,8 @@ var (
 	logLevel   string
 )
 
-const checkCmdFlushInterval = 10000000000
+// Make the check cmd aggregator never flush by setting a very high interval
+const checkCmdFlushInterval = 1000000000 * time.Second
```

Comment:
do we need such a big interval? Can `time.Hour` be enough?

Reply:
time.Hour would be enough you're right, I'll change that